tests/unit/models/transactions/test_permissioned_domain_delete.py

```diff
@@ -24,7 +27,9 @@ def test_domain_id_too_short(self):
 
         self.assertEqual(
             err.exception.args[0],
-            "{'PermissionedDomainDelete': 'domain_id must be 64 characters long.'}",
+            "{'PermissionedDomainDelete': 'domain_id must be "
+            + str(DOMAIN_ID_LENGTH)
+            + " characters long.'}",
```

Comment:
You can still use what @mvadari recommended:
```suggestion
            f"{{'PermissionedDomainDelete': 'domain_id must be {DOMAIN_ID_LENGTH} "
            "characters long.'}}",
```

Reply:
how is this better than what I already have in the code?

Reply:
You don't think it's cleaner? f-strings are generally considered the best practice for string interpolation. It's why we also use them in the codebase too.

Reply:
ok
[8206d73](https://github.com/XRPLF/xrpl-py/pull/773/commits/8206d73d87680dd34b3a008857a439eece88405d)